tmdrv_devices/thrustmaster_tmx.py

```diff
@@ -0,0 +1,9 @@
+name = 'Thrustmaster TMX'
+idVendor = 0x044f
+idProduct = [0xb67e, 0xb65d, 0xb67f]
+control = [
+	{'step':1, 'request_type':0x41, 'request':83, 'value':0x0001, 'index':0x0000, 'data':b''},
+	{'step':2, 'request_type':0x41, 'request':83, 'value':0x0007, 'index':0x0000, 'data':b''},
+]
+jscal = '6,1,0,32505,32505,16574,2049063,1,0,519,519,1036399,1067305,1,0,512,512,1050596,1054724,1,0,511,511,1050596,1048544,1,0,0,0,536854528,536854528,1,0,0,0,536854528,536854528'
+dev_by_id = 'usb-Thrustmaster_Thrustmaster_TMX_Racing_Wheel-joystick'
```

Comment:
Just after a fresh hotplug it doesn't even show up in by-id until tmdrv is run. Perhaps jscal calibration is running before the symlink is made?

-----------
blip@Terminus:\~/git/tmdrv$ ls /dev/input/by-id/
usb-046d_Logitech_Webcam_C930e_35FD9CCF-event-if00
usb-Blue_Microphones_Yeti_Stereo_Microphone_REV8-event-if03
usb-Corsair_Corsair_K70R_Gaming_Keyboard-event-if01
usb-Corsair_Corsair_K70R_Gaming_Keyboard-event-kbd
usb-Corsair_Corsair_K70R_Gaming_Keyboard-if02-event-kbd
usb-Logitech_USB_Receiver-if02-event-mouse
usb-Logitech_USB_Receiver-if02-mouse
usb-Valve_Software_Steam_Controller-event-mouse
usb-Valve_Software_Steam_Controller-mouse
blip@Terminus:\~/git/tmdrv$ ./tmdrv.py -d thrustmaster_tmx
jscal: can't open joystick device: No such file or directory
jscal non-zero exit code 1, device may not be calibrated
blip@Terminus:\~/git/tmdrv$ ls /dev/input/by-id/
usb-046d_Logitech_Webcam_C930e_35FD9CCF-event-if00
usb-Blue_Microphones_Yeti_Stereo_Microphone_REV8-event-if03
usb-Corsair_Corsair_K70R_Gaming_Keyboard-event-if01
usb-Corsair_Corsair_K70R_Gaming_Keyboard-event-kbd
usb-Corsair_Corsair_K70R_Gaming_Keyboard-if02-event-kbd
usb-Logitech_USB_Receiver-if02-event-mouse
usb-Logitech_USB_Receiver-if02-mouse
usb-Thrustmaster_Thrustmaster_TMX_Racing_Wheel-event-joystick
usb-Thrustmaster_Thrustmaster_TMX_Racing_Wheel-joystick
usb-Valve_Software_Steam_Controller-event-mouse
usb-Valve_Software_Steam_Controller-mouse